depth_first_traversal/DepthFirstTraversal.py

```diff
@@ -0,0 +1,51 @@
+def depthFirstTraversalRecursive(graph, node, visited):
+    if node not in visited:
+        visited.append(node)
+        childs = graph[node]
+        if len(childs) > 0:
+            for child in childs:
+                depthFirstTraversalRecursive(graph, child, visited)
+    return visited
+
+
+def depthFirstTraversalIterative(graph, node):
+    visited = []
+    stack = []
+    stack.append(node)
+    while len(stack) > 0:
+        node = stack[-1]
+        if node not in visited:
+            visited.append(node)
+            childs = graph[node]
+            while len(childs) > 0:
+                # add the non-visited childs to the stack in reverse order
+                child = childs.pop()
+                if child not in visited:
+                    stack.append(child)
+        else:
+            stack.pop()
+    return visited
+
+
+def main():
+    graph = {'A': ['B', 'C', 'E'],
+             'B': ['D', 'F'],
+             'C': ['G'],
+             'D': [],
+             'E': ['F'],
+             'F': ['C'],
+             'G': []}
+
+    print "Recusrive:"
```

Comment:
Please use parentheses with `print`. The code needs to be `Python3` compatible too.